src/catalog/catalog.cpp

```diff
@@ -297,6 +330,21 @@ std::string Catalog::genSerialName(const std::string& tableName, const std::stri
     return std::string(tableName).append("_").append(propertyName).append("_").append("serial");
 }
 
+bool Catalog::isInternalSequence(transaction::Transaction* transaction, const std::string& name) {
```

Comment:
I forgot how we decide to define internal sequence? Is it naming based?

Reply:
it is naming based

Reply:
Created a new catalogSet: internalSequence